graph/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM node:16.15
+
+WORKDIR /app
+
+COPY package.json yarn.lock ./
+RUN yarn
```

```suggestion
RUN yarn install --production --frozen-lockfile
```

Strictly speaking, I'd prefer pnpm like in the rest of this repo, but you'd have to get that into the container. If you've got a bit of free time, could you [add pnpm using corepack](https://pnpm.io/installation#using-corepack) and use it instead of yarn?

---

Will check it out, never used pnpm before tbh